src/components/Avatar/Avatar.styles.tsx

```diff
@@ -1,12 +1,20 @@
-import styled from 'styled-components';
+import styled, { css } from 'styled-components';
 import fonts from '../../styles/fonts';
 import color from '../../styles/colors';
 import resetCSS from '../../styles/reset';
+import { AvatarProps } from './types';
 
-interface IAvatarStyeldProps {
-    rounded: boolean;
-}
-const DivStyled = styled.div<IAvatarStyeldProps>`
+type TStyleProps = Pick<AvatarProps, 'image' | 'isRounded' | 'theme'>;
+
+export const roundedEdgeValue = '20px';
+
+const customBackgroundImage = css<TStyleProps>`
+    background-position: center;
+    background-size: cover;
+    background-image: url(${(p) => p.image});
+`;
+
+const DivStyled = styled.div<TStyleProps>`
```

Comment:
I guess we need to make the `display: inline;` rule for the element because usually, we use text in the same block.

![image](https://user-images.githubusercontent.com/78314301/151702662-35f599d3-318e-43dd-b9d2-66462328085d.png)